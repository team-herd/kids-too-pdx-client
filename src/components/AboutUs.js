import React from 'react';

function AboutUs() {
  return (
    <>
    <h1>About Us</h1>
    <h2>Shaba Rahavi</h2>
    <h2>Teonna Zaragoza</h2>
    <h2>Dyanna Tolman</h2>
    </>
  );
}
export default AboutUs;
