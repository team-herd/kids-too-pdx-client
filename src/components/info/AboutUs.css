.AboutUs {
  margin: auto;
  text-align: center;
  background-image: url('../../../public/assets/background1.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-attachment: fixed;
  height: 100vh;
  padding: 1em;
  p {
    padding: 1em;
  }
  h1 {
    margin: 0;
  }
}