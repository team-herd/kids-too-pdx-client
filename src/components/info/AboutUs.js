import React from 'react';
import styles from './AboutUs.css';

function AboutUs() {
  return (
    <section className={styles.AboutUs}>
      <h1>About Us</h1>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <h2>Teonna Zaragoza</h2>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <h2>Dyanna Tolman</h2>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <h2>Shaba Rahavi</h2>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
      <p>hellohellohellohellohellohellohellohellohellohello</p>
    </section>
  );
}
export default AboutUs;
