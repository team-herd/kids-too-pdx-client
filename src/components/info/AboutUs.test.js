import React from 'react';
import renderer from 'react-test-renderer';
import AboutUs from './AboutUs';

describe('AboutUs', () => {
  it('snapshots About Us', () => {
    const tree = renderer.create(
      <AboutUs />
    );

    expect(tree).toMatchSnapshot();
  });
});
