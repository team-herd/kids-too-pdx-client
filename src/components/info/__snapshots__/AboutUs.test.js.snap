// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs snapshots About Us 1`] = `
<section
  className="AboutUs"
>
  <h1>
    About Us
  </h1>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <h2>
    Teonna Zaragoza
  </h2>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <h2>
    Dyanna Tolman
  </h2>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <h2>
    Shaba Rahavi
  </h2>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
  <p>
    hellohellohellohellohellohellohellohellohellohello
  </p>
</section>
`;
